src/views/GameListView.jsx

```diff
@@ -132,9 +147,25 @@ const GameListView = () => {
       <Content>
         <Filter>
           <h2 className="incentive">{t("Start a game now")}</h2>
+          <StyledGameFilters>
+            <StyledGameFilter>
+              <StyledInput
+                type="search"
+                id="game-search"
+                name="game-search"
+                aria-label="Search for a game"
+                placeholder="Search for a game"
+                value={searchTerm}
+                onChange={handleChange}
+              />
+            </StyledGameFilter>
+          </StyledGameFilters>
+          <StyledGameResultNumber>
+            {filteredGameList.length + " games available"}
```

Comment:
You should use interpolation here.

Reply:
Also, not sure on this:

```suggestion
            {filteredGameList.length + " game availables"}
```
???

Reply:
I don't think `available` goes plural.